site/components/pages/Pledge/PledgeDashboard/Cards/AssetBalanceCard/index.tsx

```diff
@@ -16,6 +16,7 @@ import { Holding } from "components/pages/Pledge/types";
 import { BaseCard } from "../BaseCard";
 import { TokenRow } from "./TokenRow";
 import * as styles from "./styles";
+import { toNumber } from "lodash";
```

Comment:
```suggestion
import toNumber from "lodash/toNumber";
```
We can treeshake this function import to prevent the bundle from including the entire lodash module.
Also would you be able to group this import with the other lodash imports above? :)

Reply:
Thanks!
Great suggestion - I haven't come across that plugin before but can see the benefits in standardising imports across the repo. I'm not opposed to it.

@Atmosfearful what are your thoughts?

Reply:
#757